kasa/smartcam/smartcammodule.py

```diff
@@ -37,6 +35,8 @@ def query(self) -> dict:
 
         Default implementation uses the raw query getter w/o parameters.
         """
+        if not self.QUERY_GETTER_NAME:
+            return {}
```

Comment:
Feel free to add a test also for this if you wish, otherwise this is good to go!

Reply:
It’ll be covered by the HomeKit PR so I think we’re good 😄